src/smokeTest/java/uk/gov/hmcts/reform/iacaseapi/SmokeTest.java

```diff
@@ -16,20 +17,20 @@
         );
 
     @Test
-    public void should_welcome_upon_root_request_with_200_response_code() {
+    public void should_prove_app_is_running_and_healthy() {
 
         RestAssured.baseURI = targetInstance;
         RestAssured.useRelaxedHTTPSValidation();
 
-        String response = RestAssured
-            .given()
+        String response = SerenityRest
             .when()
-            .get("/")
+            .get("/health")
```

Comment:
we are already verify health as a part of the deployment.. it would be worth to check any simple get if we have any.

Reply:
Right now, we do not have a GET resource (except health) that can be accessed without security tokens -- all other application endpoints are secure.

Reply:
What we could do is remove security from this controller's endpoint:
https://github.com/hmcts/ia-case-api/blob/master/src/main/java/uk/gov/hmcts/reform/iacaseapi/controllers/RootController.java#L26
To allow it to be useful to the smoke test, but previously Rob said testing the welcome message was not a useful smoke test.